### DISCUSSION TITLE
First time - GEOS with mouse, no movement

### DISCUSSION THREAD

**Discussion by mabtn** [dirkwhoffmann/virtualc64#759] [Q&A] [answered]
Hi, I am trying to use GEOS with the mouse of my Mac with M1 chip.
What I do is to connect a mouse in port 1. Then, I see that the clicks are taken: GEOS reacts to them. But the mouse pointer does not move at all.
What am I doing clearly wrong?
Sorry for the question, but I am new to VirtualC64 and for sure I am missing something obvious.
Thanks
Marco

PS: in GEOS there is a keyboard shortcut, Commodore+I, to configure the input. I tried it with Option+I but it seems that the keyboard does work either. Without connecting the mouse, or even connecting the mouse, nothing changes.

**Comment by dirkwhoffmann** [comment, marked as answer]
You need to do two things:

- Plug a mouse into the control port:
<img width="791" alt="Bildschirm­foto 2023-02-15 um 21 00 02" src="https://user-images.githubusercontent.com/12561945/219140085-34d25e69-ee80-4fca-adc9-c85be792a4fb.png">

- Select the correct mouse model (must match what is configured in GEOS)
<img width="643" alt="Bildschirm­foto 2023-02-15 um 21 00 55" src="https://user-images.githubusercontent.com/12561945/219139906-9953c430-4156-46ae-bb46-990c8361871f.png">

**Reply by mabtn** [reply]
I had to change the mouse model to 1351, and now it works perfectly!
Thank you very much
